sys/include/net/gnrc/tcp/config.h

```diff
@@ -109,28 +109,28 @@ extern "C" {
 #endif
 
 /**
- * @brief Lower bound for RTO = 1 sec (see RFC 6298)
+ * @brief Lower bound for RTO in milliseconds. Default is 1 sec (see RFC 6298)
  *
  * @note Retransmission Timeout (RTO) determines how long TCP waits for
  * acknowledgment (ACK) of transmitted segment. If the acknowledgment
  * isn't received within this time it is considered lost.
  */
 #ifndef CONFIG_GNRC_TCP_RTO_LOWER_BOUND
-#define CONFIG_GNRC_TCP_RTO_LOWER_BOUND (1U * US_PER_SEC)
+#define CONFIG_GNRC_TCP_RTO_LOWER_BOUND (1U * MS_PER_SEC)
 #endif
 
 /**
- * @brief Upper bound for RTO = 60 sec (see RFC 6298)
+ * @brief Upper bound for RTO in milliseconds. Default is 60 sec (see RFC 6298)
  */
 #ifndef CONFIG_GNRC_TCP_RTO_UPPER_BOUND
-#define CONFIG_GNRC_TCP_RTO_UPPER_BOUND (60U * US_PER_SEC)
+#define CONFIG_GNRC_TCP_RTO_UPPER_BOUND (60U * MS_PER_SEC)
 #endif
 
 /**
- * @brief Assumes clock granularity for TCP of 10 ms (see RFC 6298)
+ * @brief Clock granularity for TCP in milliseconds. Dedault is 10 milliseconds (see RFC 6298)
  */
 #ifndef CONFIG_GNRC_TCP_RTO_GRANULARITY
-#define CONFIG_GNRC_TCP_RTO_GRANULARITY (10U * MS_PER_SEC)
+#define CONFIG_GNRC_TCP_RTO_GRANULARITY (10U)
 #endif
```

Comment:
I'm a big fan of adding units to the define.
In this case we can also make sure that we didn't miss to convert anything from µs to ms.

Reply:
Addressed [here ]( https://github.com/RIOT-OS/RIOT/pull/14500/commits/7d7a83918cbbc65ebaddd5b6cbf29cde09c865d8)